apps/src/sites/studio/pages/blockly.js

```diff
@@ -0,0 +1,2 @@
+import Blockly from '@code-dot-org/blockly';
+window.Blockly = Blockly;
```

Comment:
am I understanding correctly that this is here to make Blockly still available on the window object like it used to be, and that the next steps are to add `import Blockly` to all the code that has Blockly as a dependency, then remove this placeholder?

Reply:
That's the idea, yup.